src/main/java/hu/wurfel/new_school_reference/teacher/TeacherService.java

```diff
@@ -24,15 +24,15 @@ public Teacher toEntity(TeacherDto dto) {
 		return this.mapper.map(dto,Teacher.class);
 	}
 
-	public List<Teacher> findAllByName(String name){
-		return repo.findAllByNameAndDeletedIsFalse(name);
+	public List<Teacher> findAllByName(String name, boolean deleted){
+		return repo.findAllByNameAndDeleted(name, deleted);
 	}
 
-	public List<Teacher> findAllByBirthDate(Date date){
-		return repo.findAllByBirthDateAndDeletedIsFalse(date);
+	public List<Teacher> findAllByBirthDate(Date date, boolean deleted){
```

Comment:
And again
`findAllByBirthDateAndDeleted`

Reply:
d